urlshort/cmd/main.go

```diff
@@ -16,6 +16,8 @@ func main() {
 		"/yaml-godoc":     "https://godoc.org/gopkg.in/yaml.v2",
 	}
 	mapHandler := urlshort.MapHandler(pathsToUrls, mux)
+	// http.ListenAndServe(":8080", mapHandler)
```

Comment:
Remember to delete commented out code and unused functions!

Reply:
OK